caravel/storage/entities.py

```diff
@@ -55,6 +58,7 @@ def keywords(self):
         # Tokenize title and body (ranking them equally)
         words = self.title.split() + self.body.split()
```

Comment:
I would suggest doing 37 iff adding self.seller to words. Otherwise, someone changing fold_query_term(...) will have a sad.